modules/_labs/badge/react/lib/Badge.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import {keyframes} from '@emotion/core';
+import styled from '@emotion/styled';
+import {borderRadius, colors, fontFamily} from '@workday/canvas-kit-react-core';
+
+export type BadgeVariant = 'default' | 'inverse';
```

Comment:
Nit: If we're not doing an enum you could probably just inline this in the interface.

Reply:
Yeah, I added this for readability (and if other variants need to be added later, but I can do it inline. 👍